src/storage-operators/src/s3_oneshot_sink.rs

```diff
@@ -470,277 +486,20 @@ impl S3KeyManager {
     }
 }
 
-/// Required state to upload batches to S3
```

Comment:
this whole struct & corresponding implementation has been renamed to `PgCopyUploader` and moved to `storage-operators/src/s3_oneshot_sink/pgcopy.rs`